### PR TITLE
Documentation/code.gov registration

### PR DESCRIPTION
**Pull request by snallamothu**
# PR Details
## Description

Added code.gov registration info to readme.md

## Related Issue

Issue #372

## Motivation and Context

Required for integration with CodeHub

## How Has This Been Tested?

Yes

## Types of changes

- [ ] Defect fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that cause existing functionality to change)

## Checklist:

- [ ] I have added any new packages to the sonar-scanner.properties file
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
[CARMA Contributing Guide](Contributing.md) 
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -95,6 +95,24 @@ The development team would like to acknowledge the people who have made direct c
...
+
+Status: Beta
+
+Tags: transportation, cooperative driving automation, connected vehicles, intelligent transportation systems, C++, java, python, docker
+
+Labor hours: 200
```
Comment (TonysCousin) [inline README.md]: What does 200 represent?  I guess this is a standard code.gov template, but in this context it is mysterious.

File: README.md [hunk 2]
```diff
@@ -95,6 +95,24 @@ The development team would like to acknowledge the people who have made direct c
...
...low testing of arbitration and negotiation strategies. Ultimately, this project will enhance infrastructure efficiency and reduce traffic congestion. One of the goals of this project is to accelerate the technology transfer and deployment of cooperative vehicle applications through open source collaboration with industry and stakeholders. This project will enable Society of Engineers (SAE) Level 2 and Level 3 (L2/3) light vehicle testing capabilities and develop a fleet of four SAE L2/3 heavy vehicles. The project will also develop a high-level concept for Transportation System Management and Operations (TSMO) around the use of automated driving technology. Furthermore, this project will identify Traffic Incident Management (TIM) scenarios that provide strategy improvements and new strategies for first responder use cases interacting with ADS and using ADS. The new FHWA fleet will be used to prototype a scenario for each of the following use case categories: (1) Basic Travel, (2) Work Zone, (3) Traffic Incident Management and (4) Weather. These tests will provide objective data to inform stakeholders about the impacts of ADS and possible strategies to use ADS to improve operations.
```
Comment (TonysCousin) [inline README.md]: References to Phase I and II are not clear, but seem to correlate to CARMA2 and CARMA3. Can we just use those labels?

File: README.md (hunk 1 above)
Comment (snallamothu) [reply README.md]: The value doesn't mean anything for us,  It was in the template and I haven't checked if there was a data dictionary to understand what this is supposed to be.  May be we can delete.

File: README.md (hunk 2 above)
Comment (snallamothu) [reply README.md]: This came verbatim from the description we gave to code.gov and I think you are right about it correlating with CARMA2 and CARMA3.

**Pull request merged**